live-examples/css-examples/fonts/font-synthesis.html

```diff
@@ -39,8 +39,8 @@
 <div id="output" class="output large hidden">
     <section id="default-example" class="default-example">
       <div id="example-element" class="transition-all">
-        <p>Most Western fonts include <span class="bold">bold</span>, <span class="italic">italic</span>, and <span class="small-caps">italic</span> variants but CJK fonts tend not to.</p>
-        <p>中文排版通常不运用<span class="bold">粗体</span>或<span class="italic">斜体</span>。</p>
+        <p class="english">Most Western fonts include <span class="bold">bold</span>, <span class="italic">italic</span>, and <span class="small-caps">small-caps</span> variants but CJK fonts tend not to.</p>
```

Comment:
```suggestion
        <p class="english">Most Western fonts include <span class="bold">bold</span>, <span class="italic">italic</span>, and <span class="small-caps">small-caps</span> variants but CJK fonts tend not to.</p>
```
```suggestion
        <p class="english">This font does not include <span class="bold">bold</span>, <span class="italic">italic</span>, and <span class="small-caps">small-caps</span> variants.</p>
```

I think the original idea of this example was that the English bit was the "Western font" that does include the variants, and the Chinese bit was the other font. So when the "Western font" actually doesn't include the variants, the wording feels contradictory.

Reply:
That seems better, thank you